node/popn.go

```diff
@@ -1280,7 +1356,8 @@ func (nd *node) addRecursive(ctx context.Context, name string, file files.Node,
 		}
 		return it.Err()
 	case files.File:
-		froot, err := nd.Add(ctx, nd.tx.Store().DAG, f)
+		chunker, layout := selectDAGParams(name, f)
```

Comment:
Then here you can do:
```suggestion
		dagOpts := selectDAGOptions(name, f)
```